src/View/Helper/FormHelper.php

```diff
@@ -1008,7 +1008,8 @@ public function input($fieldName, array $options = [])
             'required' => null,
             'options' => null,
             'templates' => [],
-            'templateVars' => []
+            'templateVars' => [],
+            'optionsLabel' => true
```

Comment:
`labelOptions` then? 

(I'll make the change tomorrow and push all the other suggestions at the same time)